lib/ansible/modules/network/meraki/meraki_rest.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Kevin Breit (@kbreit) <kevin.breit@kevinbreit.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: meraki_rest
+
+short_description: Direct access to the Cisco Meraki management dashboard
+
+version_added: "2.6"
+
+description:
+    - Enables management of Cisco's Meraki line of products through their cloud based dashboard.
+    - More information about the Meraki API can be found at U(https://dashboard.meraki.com/api_docs).
```

Comment:
I would move this to the module notes.

Reply:
Is that not module notes? What is the difference between that location and module notes?

Reply:
No, you also have:
```yaml
notes:
- More information about the Meraki API can be found at...
```
And these are places at the bottom of the documentation. Since I believe people looking at the module docs are interested in: 1. What does the module do 2. How does it work 3. Where's more information

PS I do think the notes should come before the return values in order of importance, so I am going to propose to change this.